changelogs/head.asciidoc

```diff
@@ -7,6 +7,8 @@ https://github.com/elastic/apm-server/compare/7.13\...master[View commits]
 ==== Breaking Changes
 - `network.connection_type` is now `network.connection.type` {pull}5671[5671]
 - `transaction.page` and `error.page` no longer recorded {pull}5872[5872]
+- `apm-server.sampling.tail` now requires `apm-server.data_streams.enabled` {pull}5952[5952]
+- the `traces-sampled-*` data stream is now `traces-apm.sampled-*` {pull}5952[5952]
```

Comment:
Please ensure this is clearly marked as breaking change for a _beta_ product again.

Reply:
thanks for the reminder, done